CMakeLists.txt

```diff
@@ -633,6 +633,12 @@ set_package_properties(OpenSSL
 )
 
 if(OPENSSL_FOUND)
+
+	# TODO: needed until https://gitlab.kitware.com/cmake/cmake/issues/19263 is fixed
+	if(WIN32 AND OPENSSL_USE_STATIC_LIBS)
+		target_link_libraries(OpenSSL::SSL INTERFACE crypt32)
```

Comment:
Shouldn't it be OpenSSL::Crypto?

Reply:
@zeule 
No idea. This way, it works. What is the difference? If it is better that way, I could test it out I suppose.

Reply:
Ok, I have tested with `OpenSSL::Crypto` and it also works fine. I have no idea why it works _both_ ways, but I changed to `OpenSSL::Crypto` anyway because it seems to be "more correct".

Reply:
It works both ways because libtorrent links to OpenSSL::SSL, which in turn links to OpenSSL::Crypto, as you showed yourself earlier.